packages/@okta/vuepress-site/docs/reference/api/idps/index.md

```diff
@@ -1041,8 +1043,225 @@ curl -v -X POST \
 }
 ```
 
-#### Add Smart Card Identity Provider
+#### Add Apple Identity Provider
+
+Adds Apple as an IdP to your organization
+
+##### Request Example
 
+> **Note:** The key is truncated for brevity.
+
+```json
+{
+  "type": "APPLE",
+  "status": "ACTIVE",
+  "features": [],
+  "name": "Add Apple social IdP",
+  "protocol": {
+    "algorithms": {
+      "request": {
+        "signature": {
+          "algorithm": "SHA-256",
+          "scope": "REQUEST"
+        }
+      },
+      "response": {
+        "signature": {
+          "algorithm": "SHA-256",
+          "scope": "ANY"
+        }
+      }
+    },
+    "endpoints": {
+      "acs": {
+        "binding": "HTTP-POST",
+        "type": "INSTANCE"
+      },
+      "authorization": {
+        "binding": "HTTP-REDIRECT",
+        "url": "https://appleid.apple.com/auth/authorize"
+      },
+      "token": {
+        "binding": "HTTP-POST",
+        "url": "https://appleid.apple.com/auth/token"
+      },
+      "userInfo": null,
+      "jwks": {
+        "binding": "HTTP-REDIRECT"
+      }
+    },
+    "scopes": [
+      "name",
+      "email",
+      "openid"
+    ],
+    "settings": {
+      "nameFormat": "urn:oasis:names:tc:SAML:1.1:nameid-format:unspecified"
+    },
+    "type": "SAML2",
```

Comment:
It is not SAML

Reply:
My bad. Looks like we're sending incorrect & wrong data when creating Apple IDP..

Reply:
@yuliu-okta and @haishengwu-okta 

Just send me the updated examples and I'll plug them in.